sky/shell/tracing_controller.cc

```diff
@@ -108,6 +121,8 @@ void TracingController::ManageObservatoryCallbacks(bool add) {
 void TracingController::StartTracing() {
   LOG(INFO) << "Starting trace";
 
```

Comment:
The user can press the start recording button multiple times in observatory. I would rather not assert